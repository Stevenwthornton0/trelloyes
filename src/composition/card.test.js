import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './card.js';

describe('card component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer
      .create(<Card title='First Card' content='Lorem Ipsum' />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})