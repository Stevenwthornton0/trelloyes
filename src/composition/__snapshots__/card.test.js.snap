// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card component renders UI as expected 1`] = `
<div
  className="Card"
>
  <h3
    className="title"
  >
    First Card
  </h3>
  <p
    className="content"
  >
    Lorem Ipsum
  </p>
</div>
`;
